libs/ui-lib/lib/common/features/featureGate.tsx

```diff
@@ -1,9 +1,7 @@
 import React from 'react';
 
 // Must conform Unleash constants
-export type AssistedInstallerFeatureType =
-  | 'ASSISTED_INSTALLER_SINGLE_CLUSTER_FEATURE'
-  | 'ASSISTED_INSTALLER_PLATFORM_OCI';
+export type AssistedInstallerFeatureType = 'ASSISTED_INSTALLER_SINGLE_CLUSTER_FEATURE';
```

Comment:
Yes. 
From uhc-portal this is the MR:https://gitlab.cee.redhat.com/service/uhc-portal/-/merge_requests/5785
For unleash I can remove myself. When it will be in production I'll remove it.